External/FEXCore/Source/Interface/Core/LookupCache.cpp

```diff
@@ -17,6 +17,11 @@ namespace FEXCore {
 LookupCache::LookupCache(FEXCore::Context::Context *CTX)
   : ctx {CTX} {
 
+  // Setup our PMR map.
+  // BlockLinks = BlockLinks_pma.new_object<BlockLinksMapType>(/*BlockLinks_pma*/); // Doesn't work with argument?
```

Comment:
`new_object` (or rather `polymorphic_allocator::construct`) is perhaps too smart for its own good: It automatically appends the allocator to the given argument list if the constructed type itself is allocator-aware! Hence in this commented-out line, the allocator argument ends up being passed twice to the constructors, which is why you were seeing a compile-error.

https://godbolt.org/z/5aPThb578

https://en.cppreference.com/w/cpp/memory/polymorphic_allocator/construct

> If the object is of type that itself uses allocators [...] passes this->resource() down to the constructed object.

Reply:
Wow, I hate it. But at least I can understand it.
Shifted over to `new_object` now that this is understood.